lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,83 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
-  margin: 0;
-  padding-top: 20px;
+  margin: 0 0 $nx-spacing-xl 0;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
```

Comment:
Similar to my comment on the `nx-tile` footer PR, this should probably use `container-vertical` and _not_ `container-horizontal`.

Reply:
> Similar to my comment on the `nx-tile` footer PR, this should probably use `container-vertical` and _not_ `container-horizontal`.

https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/2ac5a2b44a793efa167d05a98b550e0e9e8c5f92